src/main/java/teammates/storage/api/FeedbackResponseCommentsDb.java

```diff
@@ -440,8 +443,7 @@ protected FeedbackResponseComment getEntity(FeedbackResponseCommentAttributes at
 
         return load()
                 .filter("courseId =", attributes.courseId)
-                .filter("createdAt =", TimeHelper.convertInstantToDate(attributes.createdAt))
-                .filter("giverEmail =", attributes.giverEmail)
+                .filter("feedbackResponseId =", attributes.feedbackResponseId)
                 .keys();
     }
```

Comment:
I have changed the way we check if comment is already present in Db. `InstructorFeedbackSubmitPageUiTest` was failing because some comments were randomly not getting added to Db. Upon investigating I found this function thought comment is already in Db as all the attributes were same including `createdAt`.